plugins/modules/aci_l4l7_policy_based_redirect.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_policy_based_redirect
```

Comment:
I think we should add the 1-to-1 child relationships for ip sla mon policy in the module. This would be similar [example](https://github.com/CiscoDevNet/ansible-aci/blob/727dfce05cca80dcc0b765a091913f8ff452dfea/plugins/modules/aci_access_span_src_group.py) shared before.